tests/acceptance/features/webUIPreview/imageMediaViewer.feature

```diff
@@ -0,0 +1,8 @@
+Feature: display image in media viewer on the webUI
+
+  Scenario: preview of image with media viewer is possible
+    Given user "user1" has been created with default attributes
+    And user "user1" has logged in using the webUI
+    Given user "user1" has uploaded "image" to "testavatar.jpg"
```

Comment:
```suggestion
    And user "user1" has uploaded "image" to "testavatar.jpg"
    And user "user1" has logged in using the webUI
```

- don't need 2 `Given` lines
- probably safer to do the upload before logging in, so that it will already be there on the UI